test/models/comment_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule CodeCorps.CommentTest do
+  use CodeCorps.ModelCase
+
+  alias CodeCorps.Comment
+
+  @valid_attrs %{body: "some content", markdown: "some content", state: "some content"}
```

Comment:
This looks like it it should not be valid since the `state` is not in the set of states we expect.